api/docs/v2/storefront/index.yaml

```diff
@@ -37,6 +38,11 @@ components:
     bearerAuth:
       type: http
       scheme: bearer
+    orderToken:
+      type: apiKey
+      in: header
+      description: Api key to authorize Cart / Checkout requests. You can obtain it from `/cart/create` or `/cart` endpoint - it's part of the Order response.
```

Comment:
I wouldn't call it `Api key` since it can be mistaken for one. It's just token that authorizes you to edit your order. It shouldn't be used in any action except that